live-build/config/hooks/configuration/81-upgrade-repository.binary

```diff
@@ -80,6 +80,20 @@ EOF
 #
 rename 's/\%3a/:/g' binary/packages/*.deb
 
+#
+# Copy over the generated hotfix metadata.
+#
+if [[ -z "$HOTFIX_METDATA" ]]; then
```

Comment:
Would it make sense to name this consistently with other environment variables that point to S3 artifacts.. e.g. something like `AWS_S3_URI_HOTFIX_METADATA`? we seem to use the `AWS_S3_URI_` prefix for other variables..